mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -47,6 +53,26 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
   }, [address, client, dispatch])
 
   const tokens = useTokensAPI()
+  const DFIToken = useSelector((state: RootState) => DFITokenSelector(state.wallet))
+  const DFIUtxo = useSelector((state: RootState) => DFIUtxoSelector(state.wallet))
+  const onConvertPress = (): void => {
```

Comment:
Same for this, can be done on the component level? Also onConvert and onSend, onUtxoVsTokenPress does the same thing, can be refactored to a common component

Reply:
updated